ci-operator/step-registry/optional-operators/cvp-common/claim/optional-operators-cvp-common-claim-workflow.yaml

```diff
@@ -0,0 +1,15 @@
+workflow:
+  as: optional-operators-cvp-common-claim
+  steps:
+    pre:
+    - ref: optional-operators-configure-proxy-registry
+    - ref: optional-operators-cvp-common-apply-secrets
+    - ref: optional-operators-subscribe
+    test:
+    - ref: optional-operators-cvp-common-scorecard
+    - ref: optional-operators-cvp-common-test
```

Comment:
So previously you had more steps here and then you have removed them. They were necessary, put them back. The resulting workflow should be basically `generic-claim` + everything CVP needs

Reply:
added those ones from generic claim